tv_apps/app-deck/js/app_deck.js

```diff
@@ -206,12 +208,25 @@
     },
 
     onAppUninstalled: function ad_onAppUninstalled(apps) {
-      // TODO: uninstall an app will be fixed in bug 1115633
+      var that = this;
+      var appGridElements = apps.map(this._findAppGridElement.bind(this));
+      appGridElements.forEach(function(elem) {
+        if (elem.dataset.revokableURL) {
+          URL.revokeObjectURL(elem.dataset.revokableURL);
+        }
+        // Move focus to next or previous element of `elem`, because
+        // we are going to remove `elem` from DOM tree
+        var nextFocus = elem.nextSibling || elem.previousSibling;
```

Comment:
I would suggest using `elem.nextElementSibling` and `elem.previousElementSibling` since `nextSibling` and `previousSibling` also select text nodes.